[2022-04-30 06:29:15,767] {processor.py:163} INFO - Started process (PID=15146) to work on /opt/airflow/dags/ej8.py
[2022-04-30 06:29:15,797] {processor.py:642} INFO - Processing file /opt/airflow/dags/ej8.py for tasks to queue
[2022-04-30 06:29:15,812] {logging_mixin.py:109} INFO - [2022-04-30 06:29:15,812] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/ej8.py
[2022-04-30 06:29:16,449] {logging_mixin.py:109} INFO - [2022-04-30 06:29:16,337] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/ej8.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ej8.py", line 4, in <module>
    from airflow.providers.apache.hive.operators.hive import HiveOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2022-04-30 06:29:16,504] {processor.py:654} WARNING - No viable dags retrieved from /opt/airflow/dags/ej8.py
[2022-04-30 06:29:19,535] {processor.py:171} INFO - Processing /opt/airflow/dags/ej8.py took 3.787 seconds
